tests/commit_message_hook_test.rb

```diff
@@ -0,0 +1,86 @@
+require 'minitest/spec'
```

Comment:
please change the semantics to `fulltest.somename.rb` to enter the automated test suite.

you can choose whatever name you like, githooks, commit_message. I have no preference, just that it has a prefix fulltest.